core/src/main/java/org/jdbi/v3/core/statement/SqlStatements.java

```diff
@@ -50,6 +54,7 @@ private SqlStatements(SqlStatements that) {
         this.sqlLogger = that.sqlLogger;
         this.queryTimeout = that.queryTimeout;
         this.allowUnusedBindings = that.allowUnusedBindings;
+        this.customizers = that.customizers;
```

Comment:
I think this should be a copy, not a reference. We wouldn't want changes to one jdbi context changing the behavior of another.